api/client/service/opts.go

```diff
@@ -176,61 +263,28 @@ func (m *MountOpt) Set(value string) error {
 		}
 	}
 
-	for _, field := range fields {
-		parts := strings.SplitN(field, "=", 2)
-		if len(parts) == 1 && strings.ToLower(parts[0]) == "writable" {
-			mount.Writable = true
-			continue
+	var err error
+	switch strings.ToLower(key) {
+	case "name":
```

Comment:
@dnephin That isn't the whole grammar, but that represents a fork in the grammar, at the top-level, where there was none before.

Let's keep things simple and adhere to the orderless csv syntax that we started with.